lib/src/contextual-menu/MenuItem.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "styled-components";
+import CoreTokens from "../common/coreTokens";
+import { MenuItemProps } from "./types";
+
+const MenuItem = ({ label, icon, slot, selected, onSelect }: MenuItemProps) => (
+  <Action role="menuitem" aria-selected={selected} selected={selected} onClick={() => onSelect()}>
+    <Label>
+      {icon && <Icon aria-hidden>{typeof icon === "string" ? <img src={icon} /> : icon}</Icon>}
```

Comment:
Missing alt text

Reply:
This is a hidden icon, why should we add an `alt` text?

Reply:
I thought that using an empty string for `alt` (alongiside `aria-hidden`) was a good practice in terms of accessibility even if the images were decorative.

After reading a bit about it, it is supposed to be unnecesary, the only advantage might be future additional compatibility with W3C validation tools, so it is totally up to you.

Sorry for the inconvenience.